deploy/infra/aws/tf-deploy-infra/vars/example.auto.tfvars

```diff
@@ -0,0 +1,17 @@
+#terraform apply -var-file="./vars/example.auto.tfvars"
+
+cp_account_id = "123456789123"
+cp_region = "us-east-1"
+cp_project = "Company"
+cp_name_core = "cloud-pipeline"
+cp_name_share = "cloud-pipeline-share"
+cp_env = "dev"
+cp_vpc_cidr_core = "10.10.0.0/16"
+cp_vpc_cidr_share = "10.20.0.0/16"
+cp_subnet_public_core_1 = "10.10.90.0/24"
+cp_subnet_public_share_1 = "10.20.90.0/24"
+cp_subnet_private_core_1 = "10.10.10.0/24"
+cp_subnet_private_core_2 = "10.10.20.0/24"
```

Comment:
as I understand we now don't need such values, but another one (with relative bits for cidr) is needed